### PR TITLE
Fix missing import and more pathlib mess

### PR DESCRIPTION
**Pull request by michalfabik**
See commits.

Resolves: https://github.com/abrt/retrace-server/issues/342

### CODE REVIEW

**Comment by DaveWysochanskiRH** [comment]
FYI, after this set I get this error
~~~
$ retrace-server-interact 166679543 crash
If you want to execute the command manually, you can run
Traceback (most recent call last):
  File "/usr/bin/retrace-server-interact", line 167, in <module>
    print_cmdline(cmdline)
  File "/usr/bin/retrace-server-interact", line 28, in print_cmdline
    sys.stderr.write("$ %s\n\n" % list2cmdline(cmdline))
  File "/usr/lib64/python3.6/subprocess.py", line 481, in list2cmdline
    needquote = (" " in arg) or ("\t" in arg) or not arg
TypeError: argument of type 'PosixPath' is not iterable
~~~

**Comment by DaveWysochanskiRH** [comment]
After 7b704eb retrace-server-interact works ok again.  Thanks for fixing this up!

File: src/retrace-server-reposync-faf [hunk 1]
```diff
@@ -84,11 +84,11 @@ def generate_repo(db, outputdir: Path, opsys, release, arch):
...
-        if path.endswith('.sqlite'):
+        if path.suffix == '.sqlite':
             new_path = '%s.bz2' % path
             cr.compress_file(path, new_path, cr.BZ2)
-            Path(path).unlink()
-            path = new_path
+            path.unlink()
+            path = Path(new_path)
 
         record = cr.RepomdRecord(name, path)
```
Comment (mgrabovsky) [inline src/retrace-server-reposync-faf:93]: Just to make sure: Does `RepomdRecord.__init__` accept a `Path` or does it require a `str`?

It appears that it relies on the latter: https://github.com/rpm-software-management/createrepo_c/blob/fd950310a91b10c3a02c0d3ca1640652aac3ca2d/src/python/repomdrecord-py.c#L99

**Comment by DaveWysochanskiRH** [comment]
For the most part this works.  But I found one other problem with this set when using vmware 'vmem' file, the task fails with another "argument of type 'PosixPath' is not iterable"
~~~
[2020-06-21 12:25:27] [I] Downloading remote resources
[2020-06-21 12:25:27] [I] Retrieving local file '/cores/bugzilla/regression-test/rhel7/vmware.vmem'
[2020-06-21 12:25:27] [D] File type: data
[2020-06-21 12:25:27] [D] unknown file type, unpacking finished
[2020-06-21 12:25:27] [D] Trying hardlink
[2020-06-21 12:25:27] [D] Succeeded
[2020-06-21 12:25:27] [I] Calculating md5sum
[2020-06-21 12:26:50] [I] Post-processing downloaded file
[2020-06-21 12:26:50] [D] File type: data
[2020-06-21 12:26:50] [D] unknown file type, unpacking finished
[2020-06-21 12:26:50] [D] File type: data
[2020-06-21 12:26:50] [D] unknown file type, unpacking finished
[2020-06-21 12:26:50] [I] Downloading remote resources
[2020-06-21 12:26:50] [I] Retrieving local file '/cores/bugzilla/regression-test/rhel7/vmware.vmss'
[2020-06-21 12:26:50] [D] File type: diy-thermocam raw data (lepton 2.x), scale 28263-29440, spot sensor temperature 0.000000, unit celsius, color scheme 0, calibration: offset 0.000000, slope 18179583481275050988767358746624.000000
[2020-06-21 12:26:50] [D] unknown file type, unpacking finished
[2020-06-21 12:26:50] [D] Trying hardlink
[2020-06-21 12:26:50] [D] Succeeded
[2020-06-21 12:26:50] [I] Calculating md5sum
[2020-06-21 12:26:50] [I] Post-processing downloaded file
[2020-06-21 12:26:50] [D] File type: diy-thermocam raw data (lepton 2.x), scale 28263-29440, spot sensor temperature 0.000000, unit celsius, color scheme 0, calibration: offset 0.000000, slope 18179583481275050988767358746624.000000
[2020-06-21 12:26:50] [D] unknown file type, unpacking finished
[2020-06-21 12:26:50] [D] File type: diy-thermocam raw data (lepton 2.x), scale 28263-29440, spot sensor temperature 0.000000, unit celsius, color scheme 0, calibration: offset 0.000000, slope 18179583481275050988767358746624.000000
[2020-06-21 12:26:50] [D] unknown file type, unpacking finished
[2020-06-21 12:26:50] [E] argument of type 'PosixPath' is not iterable
~~~

**Pull request merged**